sway/ipc-server.c

```diff
@@ -668,7 +668,8 @@ void ipc_client_handle_command(struct ipc_client *client) {
 		// TODO: Check if they're permitted to use these events
 		struct json_object *request = json_tokener_parse(buf);
 		if (request == NULL) {
-			client_valid = ipc_send_reply(client, "{\"success\": false}", 18);
+			const char *msg = "[{\"success\": false}]";
+			client_valid = ipc_send_reply(client, msg, strlen(msg) + 1);
```

Comment:
We don't want to send the terminating NULL byte.

Nitpick: `const char thing[] = …` is better than `const char *thing = …`.